lighthouse-cli/chrome-finder.ts

```diff
@@ -75,14 +79,59 @@ export function darwin() {
 }
 
 export function linux() {
+  let installations: Array<string> = [];
```

Comment:
add a comment explaining the 3 ways we're looking for the binary, and their order.